src/games/strategy/engine/framework/ServerGame.java

```diff
@@ -360,73 +359,45 @@ public void stopGame() {
   }
 
   private void autoSave() {
-    FileOutputStream out = null;
-    try {
+    final File f1 = new File(SaveGameFileChooser.DEFAULT_DIRECTORY, SaveGameFileChooser.getAutoSaveFileName());
```

Comment:
`SaveGameFileChooser.ensureDefaultDirExists();` needs to come before these statements